react-frontend/src/authentication/AdditionalDetailsPage.js

```diff
@@ -15,22 +16,31 @@ function AdditionalDetailsPage() {
   console.log("Email from query:", userData.email);
   console.log("Name from query:", userData.name);
 
+
+
   const [isFormValid, setIsFormValid] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
 
-  // Handle input changes
+  // dynamic styles for inputs
+  const inputStyle = (fieldValue) => ({
+    fontSize: "14px",
+    paddingTop: fieldValue.length > 0 ? "13px" : "10px",
+    paddingBottom: fieldValue.length > 0 ? "7px" : "10px",
+  });
```

Comment:
Is this to increase the input size when someone types so during the transition theres more space for the placeholders to go up? If so could we add this padding to the input-label-full css? or does that not work in the same way?

Reply:
It is to make sure the typing carat (the little bar) is aligned with the label at first and then moves to be aligned lower with where the inputted text appears.